posthog/models/event/sql.py

```diff
@@ -185,6 +185,75 @@
     )
 )
 
+
+KAFKA_EVENTS_RECENT_TABLE_JSON_SQL = lambda: (
+    EVENTS_TABLE_BASE_SQL
+    + """
+    SETTINGS kafka_skip_broken_messages = 100
+"""
+).format(
+    table_name="kafka_events_recent_json",
+    cluster=settings.CLICKHOUSE_CLUSTER,
+    engine=kafka_engine(topic=KAFKA_EVENTS_JSON, group="group1_recent"),
+    extra_fields="",
+    materialized_columns="",
+    indexes="",
+)
+
+EVENTS_RECENT_TABLE_JSON_MV_SQL = (
+    lambda: """
+CREATE MATERIALIZED VIEW IF NOT EXISTS events_recent_json_mv ON CLUSTER '{cluster}'
+TO {database}.{target_table}
+AS SELECT
+uuid,
+event,
+properties,
+timestamp,
+team_id,
+distinct_id,
+elements_chain,
+created_at,
+person_id,
+person_created_at,
+person_properties,
+group0_properties,
+group1_properties,
+group2_properties,
+group3_properties,
+group4_properties,
+group0_created_at,
+group1_created_at,
+group2_created_at,
+group3_created_at,
+group4_created_at,
+person_mode,
+_timestamp,
+_offset
```

Comment:
can we have more of the kafka virtual columns here? I'm thinking at least:
- `_topic`
- `_key` <- not really too important
- `_offset`
- `_timestamp`
- `_timestamp_ms`
- `_partition` <- We do want this in there so we can keep high water mark's by partition (otherwise pretty useless)

And especially these two could be interesting to use this table as a DLQ for broken events sorted by `_timestamp` which could come in handy
- `_raw_message`
- `_error`

Reply:
Do we really want the topic? I don't think it adds too much value, as it is always `clickhouse_events_json`.